drivers/wifi/nrf700x/osal/fw_if/umac_if/src/fmac_api.c

```diff
@@ -4274,6 +4220,55 @@ enum wifi_nrf_status wifi_nrf_fmac_rf_params_get(struct wifi_nrf_fmac_dev_ctx *f
 }
 
 
+enum wifi_nrf_status wifi_nrf_fmac_set_reg(struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx,
+					   struct wifi_nrf_fmac_reg_info *reg_info)
+{
+	enum wifi_nrf_status status = WIFI_NRF_STATUS_FAIL;
+	struct nrf_wifi_cmd_req_set_reg *set_reg_cmd = NULL;
+
+	if (!fmac_dev_ctx || !reg_info) {
+		wifi_nrf_osal_log_err(fmac_dev_ctx->fpriv->opriv,
+				      "%s: Invalid parameters\n",
+				      __func__);
+		goto out;
+	}
+
+	set_reg_cmd = wifi_nrf_osal_mem_zalloc(fmac_dev_ctx->fpriv->opriv,
+					       sizeof(*set_reg_cmd));
+
+	if (!set_reg_cmd) {
+		wifi_nrf_osal_log_err(fmac_dev_ctx->fpriv->opriv,
+				      "%s: Unable to allocate memory\n",
+				      __func__);
+		goto out;
+	}
+
+	set_reg_cmd->umac_hdr.cmd_evnt = NRF_WIFI_UMAC_CMD_REQ_SET_REG;
+	set_reg_cmd->umac_hdr.ids.valid_fields = 0;
+
+	wifi_nrf_osal_mem_cpy(fmac_dev_ctx->fpriv->opriv,
+						  set_reg_cmd->nrf_wifi_alpha2,
+						  reg_info->alpha2,
+						  NRF_WIFI_COUNTRY_CODE_LEN);
+
+	set_reg_cmd->valid_fields = NRF_WIFI_CMD_REQ_SET_REG_ALPHA2_VALID;
+
+	/* New feature in rev B patch */
```

Comment:
Is this comment needed?

Reply:
This was pre-existing comment. 
@krish2718 Please update.